### PR TITLE
supporting random creds for the Atlas operator

### PR DESCRIPTION
**Pull request by jasonmimick**
This adds support for generating a random password for the Atlas DB User CR. There is also docs on how to inject a random username into the chart.
﻿

### CODE REVIEW

File: charts/atlas-cluster/values.yaml [hunk 1]
```diff
@@ -60,7 +60,7 @@ clusters:
 users:
   - username: admin-user
     databaseName: admin
-    password: "%SomeLong%password$foradmin"
+    password:
```
Comment (rodrigovalin) [inline charts/atlas-cluster/values.yaml:63]: Maybe just remove this attribute from the file, makes it harder for users to mistakenly commit it or share

**Review by rodrigovalin — APPROVED** [review]
LGTM

File: charts/atlas-cluster/README.md [hunk 2]
```diff
@@ -49,6 +49,11 @@ helm install atlas-cluster mongodb/atlas-cluster\
     --set atlas.publicApiKey='<publicKey>' \
     --set atlas.privateApiKey='<privateApiKey>'
 ```
+Note, by default a random password will be generated. You can optionally also pass in a random username, however since this value is shared across templates this must be passed in, for example:
+
+```shell
+helm template --set "users[0].username=$(mktemp | cut -f2 -d.)" my-cluster mongodb/atlas-cluster
```
Comment (rodrigovalin) [inline charts/atlas-cluster/README.md:55]: This should be `helm install` instead of `helm template` right?

**Review by rodrigovalin — APPROVED** [review]
LGTM

**Comment by jasonmimick** [comment]
Fixes #83

**Pull request merged**